src/main/java/org/assertj/core/util/diff/Delta.java

```diff
@@ -136,6 +136,7 @@ int lineNumber() {
   }
 
   String formatLines(List<T> lines) {
+    //TODO see what we should use here
     return STANDARD_REPRESENTATION.format(lines, DEFAULT_START, DEFAULT_END, ELEMENT_SEPARATOR_WITH_NEWLINE, "   ");
```

Comment:
I moved the `toString` logic from the `Delta` and its subclasses in `StandardRepresentation`, now `Delta.toString()` is simply a call to `ConfigurationProvider.CONFIGURATION_PROVIDER.representation().toStringOf(this);`